### PR TITLE
Edit docs for Table

### PR DESCRIPTION
**Pull request by kennyt**
* put infinite scroll table within a container to not make the docs hard to navigate
* have VirtualList setState in `willMount`

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

File: docs/src/Table/index.js [hunk 1]
```diff
@@ -325,6 +330,7 @@ class TableExample extends React.Component {
     // The first column will be a "name" column.
     {
       className: 'name',
+      dontCache: true,
```
Comment (daiyiwhale) [inline docs/src/Table/index.js]: Why aren't we caching in the example?

Comment (kennyt) [inline docs/src/Table/index.js]: done

**Comment by daiyiwhale** [comment]
looks good!

**Pull request merged**